docs/guides/troubleshooting.md

```diff
@@ -0,0 +1,53 @@
+# Troubleshooting
+
+## Problems with media formats
+Make sure you are using formats that video.js can play:
+* Does your browser/OS support the type of video that you are trying to play?
+* Do you have a plugin that will add support for that video format?
```

Comment:
Maybe link to videojs-contrib-hls/dash and videojs-youtube as non-exhaustive examples.